srv/pillar/ceph/rgw.sls-example

```diff
@@ -0,0 +1,23 @@
+
+
+#rgw_configurations:
+#  rgw:
+#    users:
+#      - { name: "demo", display: "Demo", email: "demo@demo.nil" }
+#      - { name: "demo1", display: "Demo1", email: "demo1@demo.nil" }
+
+rgw_configurations:
+  silver:
+    users:
+      - { name: "demo", display: "Demo", email: "demo@demo.nil" }
```

Comment:
we'll rename the dict as {uid: "demo", "name": "Demo", email: "foo@bar.com"}? so as to align more with rgw's own convention, also uid and email has to be unique, name not really